taxonomy-editor-frontend/src/pages/startproject/index.jsx

```diff
@@ -0,0 +1,100 @@
+import { Typography, Box, Grid, TextField, Stack, Autocomplete, Snackbar, Alert } from "@mui/material";
+import LoadingButton from '@mui/lab/LoadingButton';
```

Comment:
I guess if this is the only use for it, I'd prefer not to have an additional dependecy. Alternatively we could show a spinner in the button, or disable it while loading.

Reply:
Yep, I created my own function `LoadingButton` to replace this